prov/verbs/src/verbs_info.c

```diff
@@ -732,22 +729,12 @@ static int vrb_have_device(void)
 	return ret;
 }
 
-static void vrb_alter_hmem_attrs(struct fi_info *info)
+static bool vrb_hmem_supported(const char *dev_name)
 {
-	if (vrb_gl_data.peer_mem_support &&
-	    strstr(info->nic->device_attr->name, "mlx")) {
-			VERBS_INFO(FI_LOG_CORE, "HMEM supported for %s\n",
-				   info->nic->device_attr->name);
-			return;
-	}
-
-	info->caps &= ~FI_HMEM;
-	info->tx_attr->caps &= ~FI_HMEM;
-	info->rx_attr->caps &= ~FI_HMEM;
-	info->domain_attr->mr_mode &= ~FI_MR_HMEM;
+	if (vrb_gl_data.peer_mem_support && strstr(dev_name, "mlx"))
+		return true;
 
-	VERBS_INFO(FI_LOG_CORE, "HMEM not supported for %s\n",
-		   info->nic->device_attr->name);
+	return false;
```

Comment:
Same as above, this is undoing a change added by a prior patch.  Update the first patch to have a clean series to apply.  That's needed in order to easily cherry pick fixes for backporting.

Reply:
Will fix.